File: API/Dtos/Author/BookAuthorDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using API.Dtos.BaseEntity;

namespace API.Dtos.Author
{
    public class BookAuthorDto  : BaseEntityDto
    {
       
        public int? AuthorId{get;set;}
        public string? AuthorName{get;set;}
        public string? BookName{get;set;}
        public int? BookId {get;set;}
    }
}

File: UI/src/app/dtos/Autors/bookAuthorDto.ts
import { BaseEntityDto } from "../base/baseEntiteDto";
import { BookDto } from "../Book/bookDto";
import { AuthorDto } from "./authorDto";

export class BookAuthorDto extends BaseEntityDto {
        authorId: number | null;
        authorName: string;
        bookName: string;
        bookId: number | null;
    }

